R/bootstrap-layout.R

```diff
@@ -384,6 +393,8 @@ verticalLayout <- function(..., fluid = TRUE) {
 #' @param cellArgs Any additional attributes that should be used for each cell
 #'   of the layout.
 #'
+#' @family layout functions
+#'
 #' @seealso [verticalLayout()]
```

Comment:
This `@seealso` section should be removed as the link is now rendered redundant.